modules/cli-options/src/main/scala/scala/cli/commands/FmtOptions.scala

```diff
@@ -42,7 +46,9 @@ final case class FmtOptions(
     scalafmtArg: List[String] = Nil,
 
   @Group("Format")
-    dialect: Option[String] = None
+    scalafmtDialect: Option[String] = None,
+  @Group("Format")
```

Comment:
NIT while you're at it, a help message would be cook here as well. and potentially a shorter alias.
```suggestion
  @Group("Format")
  @HelpMessage("Pass scalafmt version before running it. This overrides whatever value is configured in the .scalafmt.conf file.")
  @Name("fmtVersion")
```